src/test/kotlin/no/nav/familie/ef/mottak/task/OppdaterJournalføringTaskTest.kt

```diff
@@ -70,10 +73,10 @@ internal class OppdaterJournalføringTaskTest {
 
         assertThat(journalpostIdSlot.captured).isEqualTo(JOURNALPOST_DIGITALSØKNAD)
         assertThat(oppdaterJournalpostRequestSlot.captured.sak).isEqualTo(Sak(fagsakId = infotrygdSaksnummer,
-                                                                              fagsaksystem = INFOTRYGD,
+                                                                              fagsaksystem = Fagsystem.IT01,
```

Comment:
IT01 >.< 
Vet att det er verdiet for infotrygd, men `INFOTRYGD("IT01")` hadde vært enklere å skjønne :D (vet att du ikke endret denne)